python/ittapi/__init__.py

```diff
@@ -7,10 +7,13 @@
 
 from ittapi.native import Domain, Id, StringHandle
 from ittapi.native import task_begin, task_end, task_begin_overlapped, task_end_overlapped
+from ittapi.native import pt_region_begin, pt_region_end
+from ittapi.compat import pt_region_create
```

Comment:
we don't need `pt_region_create` in main module.
need to include only `pt_region_begin` and `pt_region_end`

Reply:
I have removed this now from the main module. Please have a look.